server/bin/state/test-12.reset

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+rm ${_testroot}/agent-profile
+exit ${?}
```

Comment:
FWIW, the last line doesn't accomplish anything here:  `${?}` is the default for the value used by `exit` if you omit the argument, and the default value for the script invocation is the value of the last executed command, which would be `rm` (i.e., `${?}`) if you omitted the `exit`....

Ditto `server/bin/test-bin/pbench-results-push`.